src/constants/action_types.js

```diff
@@ -0,0 +1,2 @@
+export const GET_AUTH_TOKEN_STARTED = 'GET_AUTH_TOKEN_STARTED';
```

Comment:
This should probably be `AUTHENTICATION_START`, `AUTHENTICATION_SUCCESS`, and `AUTHENTICATION_FAIL`. (`AUTH_XXX` would also be acceptable).

https://github.com/lbryio/lbry-app/blob/171ea2b97b6a393c880995a42cd4189cdda3e5bd/src/renderer/constants/action_types.js#L2